### ISSUE TITLE
Animation callbacks not called on Flipped components

### ISSUE THREAD

**Issue by christiancho** [aholachek/react-flip-toolkit#97]
Hey - thanks for a great library. I'm having an issue having the animation callbacks being called:

```
    <Flipper
      flipKey={`${lastFetch}-${type}`}
      spring="noWobble"
      handleEnterUpdateDelete={simultaneousAnimations}
      element="ul"
    >
      {extracts.map(extract => (
        <Flipped
          flipId={`${extract.id}-${type}`}
          key={`${extract.id}-${type}`}
          onAppear={() => console.log('appear')}
          onStart={() => console.log('start')}
          onStartImmediate={() => console.log('start immedate')}
          onSpringUpdate={() => console.log('spring update')}
          onComplete={() => console.log('complete')}
          onExit={() => console.log('exit')}
        >
          <Label className={type.toLowerCase()}>
            <aside>{type}</aside>
            <Link to={extract.link}>{extract.name}</Link>
          </Label>
        </Flipped>
      ))}
    </Flipper>
```
None of those console log statements are being made, but `handleEnterUpdateDelete` is being called:
```
const simultaneousAnimations = ({
  hideEnteringElements,
  animateEnteringElements,
  animateExitingElements,
  animateFlippedElements
}) => {
  console.log('animate!');
  hideEnteringElements();
  animateExitingElements();
  animateFlippedElements();
  animateEnteringElements();
};
```

Just for full context: `Label` is a styled component that receives all props. I've also verified that the `FlipContext.Consumer` does get the props correctly:
![Screen Shot 2019-10-18 at 6 40 05 PM](https://user-images.githubusercontent.com/13627157/67136120-ce6c2500-f1d6-11e9-861f-3e7615e2f71a.png)

**Comment by aholachek** [comment]
Hi, thanks for making this issue. I double checked [the handle enter/update/delete](https://codesandbox.io/s/4q7qpkn8q0) example and the basic functionality seems to be working fine (you can see the `onAppear` and `onExit` callbacks working in that example) so there's no obvious bug from that side.

Can you confirm that the `flipKey` is getting updated at the correct time, and that the FLIP transitions themselves are actually happening, the callbacks just aren't working? If it would be possible to put an example into a Codesandbox, that would also be helpful.

**Comment by christiancho** [comment]
The flipKey is definitely updating correctly, but I don't see any transitions being applied.

**Comment by christiancho** [comment]
Did a bit more digging: `animateFlippedElements ` is returning `undefined`. The docs say it should be returning a promise.

**Comment by aholachek** [comment]
Ah, it does seem like something is wrong then, I will take a look this weekend, thank you for flagging.

**Comment by christiancho** [comment]
@aholachek Is there a chance that it's a problem with object reference? I recently changed some code to return a new array of newly constructed objects on every fetch. Anyway, thanks for looking into this!

**Comment by dimensi** [comment]
In my case animation callbacks didn't fire, when inverseFlipId prop setted.

**Comment by aholachek** [comment]
> Edit: Also, passing `debug` as a prop to `Flipper` doesn't do anything. Should it be triggering the browser debugger?

No, the `debug` prop just pauses the animation at the beginning: https://github.com/aholachek/react-flip-toolkit#advanced-props

@christiancho could you just confirm you're using the latest version? (7.0.6)

**Comment by christiancho** [comment]
@aholachek Good to know about `debug`.

Yes, I'm on `7.0.6`.

**Comment by aholachek** [comment]
Hi @christiancho, thanks for confirming. I thought I might have some idea of what could be going wrong, but so far I've been unable to reproduce this issue locally. If you could make a minimal codesandbox example for me to look at, that would be extremely helpful.

**Comment by raphaelaleixo** [comment]
Hey @christiancho , @aholachek : I'm getting this same problem. Either of you were able to find out what's happening?

**Comment by cervonwong** [comment]
Same issue here.. any updates?
